eos-updater-prepare-volume/eos-updater-prepare-volume

```diff
@@ -90,7 +90,7 @@ class VolumePreparer:
         if collection_binding is None:
             return None
         else:
-            return str(collection_binding)
+            return collection_binding.get_string()
```

Comment:
I don't understand how this is necessary. I tested this and it got the collection ID correctly, which you can see here: https://phabricator.endlessm.com/F33127

But if `get_string()` works that's fine. No need to spend extra effort figuring out why the unit test environment is different than the test I did.

Reply:
:man_shrugging: I definitely got one of the subprocess arguments being `'"some-collection-binding"'`.